cypress/integration/form-builder/conditionalFieldset.test.js

```diff
@@ -0,0 +1,384 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import sanityClient from '@sanity/client'
+
+let testDocumentId = ''
+const testSanityClient = sanityClient({
```

Comment:
Is it possible to move this client configuration outside of this specific test, so that it can be reused for other tests in the future?

Reply:
Think so! I'll change it 👍